frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.scss

```diff
@@ -0,0 +1,3 @@
+.co-vm-details-cdroms__datalist-dd {
```

Comment:
dont we prefix all with css classes with `kubevirt` instead of `co` ?

Reply:
ya I was asking about that on slack yesterday, not sure what the move forward is. If we stick with Kubevirt I'd vote for `kv-` for brevity.

Reply:
yes we do ... in this case i copy pasted from here https://github.com/openshift/console/pull/3662#discussion_r353889666 , I assumed it's ok ...

Reply:
This is why I was asking because I couldn't find any docs on the right classes for kubevirt. I don't feel strongly about `co-` I just think consistency is best.